src/managers/ProjectManager.ts

```diff
@@ -601,7 +601,19 @@ export class ComponentLibraryProject extends Project {
             ],
             from: /uri\s*=\s*"(.+)\.brs"/gi,
             to: (match) => {
-                return match.replace('.brs', this.postfix + '.brs');
+
+                // do not alter file ending if it's an external library eg with a common:/ file path
+                let fileMatch = true;
+                if (match.includes('common:/')) {
+                    fileMatch = false;
+                }
+
+                if (fileMatch) {
+                    return match.replace('.brs', this.postfix + '.brs');
+                } else {
+
+                    return match
+                }
```

Comment:
Looks pretty good. I'm thinking this could be simplified a bit, like so:

```suggestion
                // do not alter file ending if it's an external library eg with a common:/ file path
                if (match.includes('common:/')) {
                    return match;
                } else {
                    return match.replace('.brs', this.postfix + '.brs');
                }
```

Reply:
OK I will take a look Bron, thanks.  Looks good :)